### PR TITLE
commit with adding files

### PR DESCRIPTION
**Pull request by ArtemovychMaria**
Added first Java Advanced homework

### CODE REVIEW

File: src/test/java/lesson1/SalaryTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    private Salary salary;
+
+    @Before
+    public void setup(){
+        salary=new Salary();
+    }
+
+    @Test
+    public void itCalculateMonthSalary() {
+        int actual=salary.calculateMonthSalary(-200,168);
```
Comment (VolodymyrTymtsiv) [inline src/test/java/lesson1/SalaryTest.java]: We need to use `@Test(expected= MyException.class)` to test unhappy cases.

Comment (ArtemovychMaria) [reply src/test/java/lesson1/SalaryTest.java]: changes added

File: src/test/java/lesson1/SalaryTest.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    private Salary salary;
+
+    @Before
+    public void setup() {
+        salary = new Salary();
+    }
+
+    @Test(expected = MyException.class)
+    public void itCalculateMonthSalary() throws MyException {
```
Comment (VolodymyrTymtsiv) [inline src/test/java/lesson1/SalaryTest.java]: The name bot actually tells what it tests.
I would recommend rename to -> `itThrowsExceptionWhenSalaryInHourIsNegative`

File: src/main/java/lesson1/MyException.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package lesson1;
+
+public class MyException extends Exception {
```
Comment (VolodymyrTymtsiv) [inline src/main/java/lesson1/MyException.java]: We could extend from `RuntimeException` that allows us to nut add `throws MyException` to methods.

**Comment by VolodymyrTymtsiv** [comment]
Looks good in general, added a few little comments :)

File: src/main/java/lesson1/MyException.java (hunk 3 above)
Comment (ArtemovychMaria) [reply src/main/java/lesson1/MyException.java]: Added new changes

**Comment by VolodymyrTymtsiv** [comment]
One more thing, please, add comments with message that describe what you did instead of "third commit" 😉

**Pull request merged**